### ISSUE TITLE
iOS crash after update 2.5.0

### ISSUE THREAD

**Issue by lDemurl** [martijn00/LottieXamarin#140]
I run my project in debug mode and throw this exception:

System.ExecutionEngineException: Attempting to JIT compile method 'Airbnb.Lottie.LOTAnimationView:PlayWithCompletion (Airbnb.Lottie.LOTAnimationCompletionBlock)' while running in aot-only mode.

**Comment by rotsen24154** [comment]
Are you using Net Standard?
I have the same problem.

**Comment by lDemurl** [comment]
Yes, I am using netstandard 2.0

**Comment by ankidave** [comment]
I'm getting a similar issue after upgrading. My error is
```
Error MT2101: Can't resolve the reference 'System.Void ObjCRuntime.BlockLiteral::SetupBlockUnsafe(System.Delegate,System.Delegate)', referenced from the method
'System.Void Airbnb.Lottie.LOTAnimationView::PlayFromFrame(Foundation.NSNumber,Foundation.NSNumber,Airbnb.Lottie.LOTAnimationCompletionBlock)'
in 'Xamarin.iOS, Version=0.0.0.0, Culture=neutral, PublicKeyToken=84e04ff9cfb79065'. (MT2101)
```

My code that produces this is as follows:
```csharp
LOTAnimationView spinner = LOTAnimationView.AnimationNamed("loaderRingSpinning");
spinner.Frame = new CGRect(new CGPoint(), SpinnerViewContainer.Frame.Size);
spinner.ContentMode = SpinnerViewContainer.ContentMode;
SpinnerViewContainer.AddSubview(spinner);
```

**Comment by mprieur18** [comment]
I've the same issue on my iOS App.
I'm not using .Net Standard ...
Below a screenshot of error logged on MS App Center :
![screen shot 2018-02-28 at 09 06 04](https://user-images.githubusercontent.com/3032969/36776782-55fdc0f6-1c67-11e8-9e2b-991c5bd8b530.png)

**Comment by fabionuno** [comment]
Are you using a device or simulator? 
What version of iOS?

**Comment by mprieur18** [comment]
Simulator and devices ...
We are using iOS 11.2

**Comment by rizaanlakay** [comment]
I had the exact same problem and was forced to remove any Lottie from my project because of deadlines. I hope you guys manage to fix this so I can add it back at some point. :)

**Comment by lDemurl** [comment]
I roll back to 2.1.5 and works fine!

**Comment by reinierkops** [comment]
Same issue here, reverted to 2.1.5 works fine

**Comment by fabionuno** [comment]
Hi @ankidave and @mprieur18 I was able to reproduce this bug setting Link behavior to **Link Framework SDKs Only**.

I built this NuGet package  [Com.Airbnb.iOS.Lottie.2.5.0.1-beta1.zip](https://github.com/martijn00/LottieXamarin/files/1772420/Com.Airbnb.iOS.Lottie.2.5.0.1-beta1.zip)  on my machine.  Please download and rename this file to `Com.Airbnb.iOS.Lottie.2.5.0.1-beta1.nupkg `

Can you check if this version fix the problem?

**Comment by matthieucolle** [comment]
your Nuget fix the problem for me. thanks !

**Comment by khle** [comment]
@fabionuno Your 2.5.0.1-beta1 works. What is the next step or ETA on the official one?

**Comment by martijn00** [comment]
@fabionuno did you change anything? What can I do?

**Comment by fabionuno** [comment]
@martijn00 Nothing was changed. I just built the project in VS4Mac

**Comment by martijn00** [comment]
@fabionuno weird, i could just compile everything again and release a new version?

**Comment by khle** [comment]
@martijn00 Could you please give that a go?

**Comment by fabionuno** [comment]
@khle We are testing a new version to release as soon as possible.

**Comment by fabionuno** [comment]
@khle I have tested the current package `Lottie iOS 2.5.0.0` with VS4Mac 7.4 and everything is working fine.
If you are using VS4Mac 7.3.3 can you update to 7.4 and check if it fix the error

**Comment by martijn00** [comment]
You can fix this by updating to VS4Mac 7.4.

**Comment by mshenoy83** [comment]
Still breaks on appcenter builds :(. I had to roll back to 2.4.1

**Comment by khle** [comment]
@mshenoy83 Does AC have 7.4 or some earlier version of VSMac ?

**Comment by mshenoy83** [comment]
![image](https://user-images.githubusercontent.com/8349984/37182343-e7ff929e-2384-11e8-9749-51f625cdebad.png)
![image](https://user-images.githubusercontent.com/8349984/37182357-f721477c-2384-11e8-9b2c-ce80a3a870f0.png)

No idea to be frank, but here are the Mono and Xcode versions being used.

**Comment by realpunk** [comment]
2.5.1 still getting 

MTOUCH : error MT2101: Can't resolve the reference 'System.Void ObjCRuntime.BlockLiteral::SetupBlockUnsafe(System.Delegate,System.Delegate)', referenced from the method 'System.Void Airbnb.Lottie.LOTAnimationView::PlayFromFrame(Foundation.NSNumber,Foundation.NSNumber,Airbnb.Lottie.LOTAnimationCompletionBlock)'

**Comment by roubachof** [comment]
same here: appcenter fails to build with 2.5.1

```
2018-03-21T10:00:22.6225950Z (_CompileToNative target) -> 
2018-03-21T10:00:22.6476210Z   MTOUCH : error MT2101: Can't resolve the reference 'System.Void ObjCRuntime.BlockLiteral::SetupBlockUnsafe(System.Delegate,System.Delegate)', referenced from the method 'System.Void Airbnb.Lottie.LOTAnimationView::PlayFromFrame(Foundation.NSNumber,Foundation.NSNumber,Airbnb.Lottie.LOTAnimationCompletionBlock)' in 'Xamarin.iOS, Version=0.0.0.0, Culture=neutral, PublicKeyToken=84e04ff9cfb79065'. 
```

**Comment by rob-derosa** [comment]
Is there a fix for this? After updating to iOS 11.2 and configuring AppCenter to use the latest Xcode and Mono builds, I'm still blocked by this issue. Other than backporting, I'm not sure what to do here. Builds work fine locally against the latest stable channel but fails in AC.

**Comment by roubachof** [comment]
We have the same issue with 2.5 as well (RELEASE on vs AC).
